### PR TITLE
Add EAN generator

### PR DESCRIPTION
**Pull request by jvanbaarsen**
Add a generator to create EAN13 codes.

### CODE REVIEW

File: test/test_code.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# encoding: utf-8
```
[the remark this answers is not in the document]
Comment (jvanbaarsen) [reply test/test_code.rb]: This is added one line below.

File: lib/ffaker/code.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# encoding: utf-8
```
[the remark this answers is not in the document]
Comment (jvanbaarsen) [reply lib/ffaker/code.rb]: Added one line below

**Comment by jvanbaarsen** [comment]
I notice that the test is failing, but this is unrelated to my change.

**Comment by marocchino** [comment]
Thanks!

**Pull request merged**